modules/codegen/src/smithy4s/codegen/Renderer.scala

```diff
@@ -362,7 +362,7 @@ private[codegen] class Renderer(compilationUnit: CompilationUnit) { self =>
               }
             )
             .block(s"$name.apply")
-            .appendToLast(if (hints.nonEmpty) ".withHints(hints)" else "")
```

Comment:
Structures should have hints on them containing the ShapeId, right?

Reply:
think so